lib/src/configuration.cc

```diff
@@ -358,25 +359,35 @@ void Configuration::defineDefaultValues()
                     Types::Int,
                     5) } });
 
+    // Hidden option: number of ping/pong timeouts to allow before disconnecting, default: 2
+    defaults_.insert(
+        Option { "allowed-keepalive-timeouts",
```

Comment:
This doesn't match how cpp-pcp-client thinks of the option, but I think it's easier to describe succinctly.